presto-native-execution/presto_cpp/presto_protocol/core/DataSize.cpp

```diff
@@ -29,16 +30,12 @@ DataSize::DataSize(const std::string& string) {
 }
 
 std::string DataSize::toString() const {
-  char buffer[32];
-  snprintf(
-      buffer,
-      sizeof(buffer),
-      "%f%s",
+  return fmt::format(
+      "{:-f}{}",
```

Comment:
Is "-" needed ? Think its the default.

Reply:
@czentgr suggested me to use {:-f} citing :  https://fmt.dev/10.2.0/syntax.html

Reply:
@anandamideShakyan : Agree with the use of {:-f} syntax. As per the examples in https://fmt.dev/10.2.0/syntax.html#format-examples, the leading "-" is not needed. It is the default. So I was suggesting that we can skip that sign.